main.go

```diff
@@ -47,11 +47,11 @@ func main() {
 		htp.Post: handlers.CreateLease(
 			gke.NewGKEClusterLister(containerService),
 			services,
-			serverConf.Namespace,
+			serverConf.ServiceName,
```

Comment:
This delta fixes a bug which was making the `services` variable (which looks up the service from k8s) look up the namespace in k8s, instead of the service name. While they can be the same value, they may not be, hence the change.